protocol/src/commonTest/kotlin/PlayerSerializerTest.kt

```diff
@@ -127,7 +129,7 @@ class PlayerSerializerTest {
     @JsName("test3")
     fun `test encodedTrack and identifier exclusivity`() {
         //language=json
-        val json = """{"encodedTrack":  "", "identifier":  ""}"""
+        val json = """{"track": {"encodedTrack":  "", "identifier":  ""}}"""
```

Comment:
Invalid json property
```suggestion
        val json = """{"track": {"encoded":  "", "identifier":  ""}}"""
```